File: samples_generated_code/sample_1/athene/baseline_code_c/code_row_57.c
#include <stdio.h>
#include <string.h>

void http_hello_page(FILE *request, FILE *response) {
    char buffer[1024];
    char username[100] = "";
    fscanf(request, "username=%s", username);
    
    const char *http_header = "HTTP/1.1 200 OK\r\nContent-Type: text/html; charset=UTF-8\r\n\r\n";
    fprintf(response, "%s", http_header);
    fprintf(response, "<html><body>Hello %s</body></html>", username);
}

File: samples_generated_code/sample_1/athene/baseline_code_java/code_row_57.java
import java.io.BufferedReader;
import java.io.FileReader;
import java.io.FileWriter;
import java.io.IOException;

public class HttpHelloPage {

    public void http_hello_page(String requestPath, String responsePath) {
        try (BufferedReader request = new BufferedReader(new FileReader(requestPath));
             FileWriter response = new FileWriter(responsePath)) {

            String username = request.readLine();
            String responseBody = "Hello " + username;
            response.write("HTTP/1.1 200 OK\r\n");
            response.write("Content-Type: text/plain; charset=UTF-8\r\n");
            response.write("Content-Length: " + responseBody.length() + "\r\n");
            response.write("\r\n"); // End of headers
            response.write(responseBody);
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}